instrumentation/opentelemetry-instrumentation-requests/tests/test_requests_integration.py

```diff
@@ -472,3 +474,49 @@ def perform_request(url: str, session: requests.Session = None):
         request = requests.Request("GET", url)
         prepared_request = session.prepare_request(request)
         return session.send(prepared_request)
+
+
+class TestRequestsIntergrationMetric(TestBase):
+    URL = "http://httpbin.org/status/200"
+
+    def setUp(self):
+        super().setUp()
+        self.reader = InMemoryMetricReader()
+        self.meter_provider = MeterProvider(metric_readers=[self.reader])
```

Comment:
Just the like the tracer provider, the meter provider with inmemory reader is added to [`TestBase`](https://github.com/open-telemetry/opentelemetry-python/blob/main/tests/opentelemetry-test-utils/src/opentelemetry/test/test_base.py#L39-L49) since this will be common. You can make use of it `memory_metrics_reader` to get metrics data.

Reply:
Done